theano/tensor/tests/test_basic.py

```diff
@@ -6863,30 +6877,31 @@ def test_infer_shape(self):
         #                        [Reshape(ndim)(adtens4, aivec)],
         #                        [adtens4_val, [1, 3, 10, 4]], Reshape)
 
-        # Tile
-        advec = dvector()
-        advec_val = rand(5)
-        aivec_val = [3]
-        ndim = 1
-        self._compile_and_check([advec],
-                                [tile(advec, aivec_val, ndim)],
-                                [advec_val], Tile)
-
-        admat = dmatrix()
-        admat_val = rand(2, 4)
-        aivec_val = [2, 3]
-        ndim = None
-        self._compile_and_check([admat],
-                                [tile(admat, aivec_val)],
-                                [admat_val], Tile)
-
-        adtens4 = dtensor4()
-        adtens4_val = rand(2, 4, 3, 5)
-        aivec_val = [2, 3, 1, 4]
-        ndim = 4
-        self._compile_and_check([adtens4],
-                                [tile(adtens4, aivec_val, ndim)],
-                                [adtens4_val], Tile)
+        # Tile op is depricated
+        
+        # advec = dvector()
+        # advec_val = rand(5)
+        # aivec_val = [3]
+        # ndim = 1
+        # self._compile_and_check([advec],
+        #                         [tile(advec, aivec_val, ndim)],
+        #                         [advec_val], Tile)
+
+        # admat = dmatrix()
+        # admat_val = rand(2, 4)
+        # aivec_val = [2, 3]
+        # ndim = None
+        # self._compile_and_check([admat],
+        #                         [tile(admat, aivec_val)],
+        #                         [admat_val], Tile)
+
+        # adtens4 = dtensor4()
+        # adtens4_val = rand(2, 4, 3, 5)
+        # aivec_val = [2, 3, 1, 4]
+        # ndim = 4
+        # self._compile_and_check([adtens4],
+        #                         [tile(adtens4, aivec_val, ndim)],
+        #                         [adtens4_val], Tile)
```

Comment:
As the Tile op is kept for now, you should uncomment the test and make the
test use the Tile op, not the new code.

On Sun, Jan 11, 2015 at 2:32 PM, Amjad Almahairi <notifications@github.com>
wrote:

> In theano/tensor/tests/test_basic.py
> <https://github.com/Theano/Theano/pull/2362#discussion-diff-22769241>:
>
> > +
> > +        # admat = dmatrix()
> > +        # admat_val = rand(2, 4)
> > +        # aivec_val = [2, 3]
> > +        # ndim = None
> > +        # self._compile_and_check([admat],
> > +        #                         [tile(admat, aivec_val)],
> > +        #                         [admat_val], Tile)
> > +
> > +        # adtens4 = dtensor4()
> > +        # adtens4_val = rand(2, 4, 3, 5)
> > +        # aivec_val = [2, 3, 1, 4]
> > +        # ndim = 4
> > +        # self._compile_and_check([adtens4],
> > +        #                         [tile(adtens4, aivec_val, ndim)],
> > +        #                         [adtens4_val], Tile)
>
> These tests are failing because the _compile_and_check function checks if
> the op (in this case Tile) is somewhere in the graph of the tile function.
> Since we're now not using Tile op in the tile function this test should
> fail. That's why I commented it out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/2362/files#r22769241>.
>